Romans/Roman Numerals Encoder/index.js

```diff
@@ -0,0 +1,15 @@
+function solution(number){
+    let finalRomanNumeral = '',
+         numbers = [1000, 900, 500, 400, 100, 90, 50, 40, 10, 9, 5, 4, 1],
+         romans    = ['M', 'CM', 'D', 'CD', 'C', 'XC', 'L', 'XL', 'X', 'IX', 'V', 'IV', 'I'];
```

Comment:
I would suggest to use prettier for better indentation of your codes
```suggestion
         romans = ['M', 'CM', 'D', 'CD', 'C', 'XC', 'L', 'XL', 'X', 'IX', 'V', 'IV', 'I'];
```